improver/calibration/utilities.py

```diff
@@ -272,3 +272,76 @@ def merge_land_and_sea(calibrated_land_only, uncalibrated):
         mask = calibrated_land_only.data.mask
         new_data[mask] = uncalibrated.data[mask]
         calibrated_land_only.data = new_data
+
+
+def forecast_coords_match(first_cube, second_cube):
+    """
+    Determine if two cubes have equivalent forecast_periods and that the hours
+    of the forecast_reference_time coordinates match. Only the points of the
+    forecast period and forecast reference time coordinate are checked. This is
+    to ensure that an EMOS coefficient cube matches the forecast cube, as
+    appropriate.
```

Comment:
Could (not important at all) remove the specific EMOS reference here now that this has been taken out as a general calibration utility.

Reply:
Let’s have a chat about this.